docs/writing/posts/learning.md

```diff
@@ -0,0 +1,153 @@
+
+### Just be ahead a little bit
+
+I quickly learned that if I spent the first few weeks working two or three times harder than everyone else to learn new software like Adobe Photoshop or Illustrator, I would be far ahead of the class. This boosted my self-esteem and made me feel intelligent. Moreover, it allowed me to gain more practice. Even if neither of us understood what was going on, the fact that someone could ask me a question and I could figure out the answer faster meant that I got to learn and teach simultaneously. This widened the gap between me and my peers significantly, which was basically two reps. Eventually, helping others just meant it gave me access to problems I wouldn't have otherwise encountered.
+
+!!! note "Something I noticed"
+
+    I was never the best artist, and I don't think that was the goal for me either. However, I noticed that most of my skilled friends were always drawing and practicing. Even to this day, there are people I know whom I have never seen without a sketchbook. They probably fill 12 to 15 sketchbooks every year. It's always amusing when people admire their work and call them talented, unaware that for the past 12 years, every time I've been late for a coffee, every time we've visited a gallery, every time we've gone for a walk, they have always been drawing.
+
+## Create your peer group
+
+After my second year of college, I decided to learn machine learning and leave physics behind. I had completed Andrew Ng's machine learning course on Coursera, and it amazed me with the possibilities. However, I didn't know anyone who shared my interest in machine learning. So, just like in high school, I pushed myself hard to get ahead.
+I founded a machine learning group called Data Hackers. I also started a data science and machine learning club at Waterloo. I even created a guide on how to start your own data science/machine learning club. In just two years, I had a facebook group of 7000 people who shared my passion for machine learning and data science.
+
+Similar to how I gained a following on Twitter, I would read papers and shared summaries. I posted helpful content. I attended hackathons and met other data science club founders to exchange ideas and support each other. I told myself that if I could make Waterloo the top data science school in Canada, my own data science skills would get pretty good.
+
+### You only need to be a little bit ahead
+
+I found myself in a situation where hundreds of people messaged me with their machine learning and data science questions, not knowing I literally no better than them. However, I would diligently search Google and YouTube to find answers and call them to discuss the topics. Just like in highschool, although I was only slightly more knowledgeable than them, having this audience allowed me collect interesting questions and find answers.
```

Comment:
I found myself in a situation where hundreds of people messaged me with their machine learning and data science questions, not knowing I literally know no better than them.